### ISSUE TITLE
Z-index is ooooover 9000!!!

### ISSUE THREAD

**Issue by leonheess** [dieparteidiepartei/afd-blocker-plugin#54]
No, but for real: The injected CSS includes `z-index: 99998;` which is, of course, a bad idea as the content that is being blocked isn't necessarily on top all the time. Instead, the z-index should be selected dynamically based on the blocked contents z-index.

Here is an easy-to-recreate example of legitimate content being obstructed:
![image](https://user-images.githubusercontent.com/13834296/60998135-a60f9780-a358-11e9-88a6-2fab4a9e813a.png)

**Comment by lennart-k** [comment]
Absolutely, in my opinion the blocked content should just get opacity: 0 and that's it.
